vagrant/roles/subman-devel/tasks/main.yml

```diff
@@ -64,6 +66,48 @@
     - libselinux-python
     - xorg-x11-server-Xvfb
     - yum-utils
+  when: ansible_distribution != 'openSUSE Leap'
+
+- name: install spacewalk repo (openSUSE Leap)
+  get_url:
+    url: http://download.opensuse.org/repositories/systemsmanagement:/spacewalk/openSUSE_Leap_42.2/systemsmanagement:spacewalk.repo
+    dest: /etc/zypp/repos.d/
+  become: yes
+
+- name: install non-pip/spec deps (openSUSE Leap)
```

Comment:
> Is the idea to leave all distro-specific non-pip and spec deps in a variable defined for each distro?

Exactly. So essentially the two tasks (`install non-pip/spec deps` and `install non-pip/spec deps (openSUSE Leap)`) will be refactored into one.